dom.bs

```diff
@@ -6341,8 +6334,8 @@ given a <var>document</var>, <var>localName</var>, <var>namespace</var>, and opt
    <var>document</var>.
 
    <li>
-    <p>If the <var>synchronous custom elements flag</var> is set, then run this step while
-    catching any exceptions:
+    <p>If <var>synchronousCustomElements</var> is true, then run this step while catching any
+    exceptions:
```

Comment:
Maybe not for this PR, but I guess we should not say anything here about exceptions, and just link to https://webidl.spec.whatwg.org/#an-exception-was-thrown somewhere in "If this step threw an exception" below. Doesn't need to happen in this PR I guess.

Reply:
Oh yes, that would be nice to adopt.